helix-term/src/commands.rs

```diff
@@ -4326,3 +4327,12 @@ fn dap_launch(cx: &mut Context) {
         let _ = block_on(request).unwrap();
     }
 }
+
+fn dap_run(cx: &mut Context) {
+    use helix_lsp::block_on;
+
+    if let Some(debugger) = &mut cx.editor.debugger {
+        let request = debugger.configuration_done();
```

Comment:
This is not always supported and you need to check the capability. It needs to be sent as the last config step before either "launch" or "continue". See the spec under "initialized" event, it explains the ordering.

I suspect what you want to do in run is either call "launch" (if unstarted) or "continue" (if previously started) to resume execution

Reply:
By standard it's not, but we're going to think about that when we find a misbehaving debugger. This request starts execution and I can't really understand how do we use debugger without that.